package.json

```diff
@@ -43,7 +43,7 @@
     "eslint-plugin-standard": "^3.0.1",
     "faker": "^3.1.0",
     "fs.extra": "^1.3.2",
-    "gulp": "^3.8.7",
+    "gulp": "^3.9.1",
```

Comment:
We'll need to check the version upgrade in a separate PR

Reply:
darn sorry! i will roll those back